playground/playground.sql

```diff
@@ -1,9 +1,17 @@
 \c nc_news_test;
 
-SELECT * FROM articles;
-SELECT * FROM comments;
-SELECT * FROM topics;
-SELECT * FROM users;
+SELECT
+articles.author, articles.article_id, articles.topic, articles.created_at,
+articles.votes, articles.article_img_url, COUNT(comments.comment_id)::INT AS comment_count
+FROM articles
+LEFT JOIN comments ON articles.article_id = comments.article_id
+GROUP BY articles.article_id
+ORDER BY articles.created_at DESC;
```

Comment:
It's great to see you've got this SQL file for playing around with query syntax to see what it does, I'd recommend adding the file names to your _.gitignore_ so they don't show up in the file changes for each PR however